packages/back-end/src/queryRunners/ExperimentResultsQueryRunner.ts

```diff
@@ -75,6 +76,13 @@ export function getFactMetricGroup(metric: FactMetricInterface) {
       return "";
     }
   }
+
+  // Quantile metrics get their own group to prevent slowing down the main query
+  if (quantileMetricType(metric)) {
+    return metric.numerator.factTableId
+      ? `${metric.numerator.factTableId} (quantile metrics)`
+      : "";
+  }
```

Comment:
Should we only move event quantiles to a separate group?  From what I understand, unit quantiles have no big performance impact on queries, right?

Reply:
They both are similarly problematic as the percentile functions themselves are likely to be expensive.

The event quantile only adds the need to join a small dimensional table (__quantileMetric, one row per variation-dimension) to a large dimensional table (__userMetricJoin one row per user).

I don't think that join should be too bad?